Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -147,7 +153,7 @@ def handle_formula_and_casks(formulae_and_casks, args)
         Repology::HOMEBREW_CASK
       end
 
-      package_data = if formula_or_cask.is_a?(Formula) && formula_or_cask.versioned_formula?
+      package_data = if skip_repology?(formula_or_cask, args:)
         nil
       else
         Repology.single_package_query(name, repository:)
```

Comment:
This makes sense but I am wondering why the logic isn't just something like this. Not related to your proposed changes though.

```rb
package_data = Repology.single_package_query(name, repository:) unless skip_repology?(formula_or_cask, args:)
```

I have not tested this locally and it may need to live on multiple lines because of the line length linter rule.

Reply:
Yeah that should work - will push.